.gitignore

```diff
@@ -1,3 +1,3 @@
-formicactl
+inagoctl
```

Comment:
can we temporariyl leave the old one in? otherwise I guarantee someone will commit his old binary.